Makefile.am

```diff
@@ -64,6 +66,7 @@ endif
 src_libfabric_la_CPPFLAGS = $(AM_CPPFLAGS)
 src_libfabric_la_LDFLAGS =
 src_libfabric_la_LIBADD =
+src_libfabric_la_DEPENDENCIES = $(srcdir)/libfabric.map
```

Comment:
That line is moved from back of the file to this place because the PSM2 changes need to add items to this variable.